docs/plugin.md

```diff
@@ -97,20 +97,48 @@ createPlugin(document.getElementById('app')!); // app is a <div> element
 To use the plugin (with the React UI) inside another React app:
 
 ```ts
-function MolStarWrapper() {
-    const parent = React.createRef<HTMLDivElement | null>();
-
-    useEffect(() => {
-        let plugin: PluginContext | undefined = undefined;
-        async function init() {
-            plugin = createPlugin(parent.current);
-        }
-        init();
-        return () => { plugin?.dispose(); };
-    }, []);
-
-    return <div ref={parent} style={{ width: 640, height: 480 }} />;
+import { useEffect, createRef } from "react";
+import { createPluginUI } from "molstar/lib/mol-plugin-ui";
+import { PluginUIContext } from "molstar/lib/mol-plugin-ui/context";
+import "molstar/build/viewer/molstar.css";
+
+declare global {
+  interface Window {
+    molstar: PluginUIContext;
+  }
 }
+
+window.molstar = window.molstar || {};
```

Comment:
This will allow to have only 1 instance of the Plugin and depending what happens to the UI could lead to incorrect state.

Reply:
I guess it naturally follows that we should add this to the docs as an extra section. :smile: 
If I can polish the details a bit and make work, I will open a new PR.

Reply:
added single-plugin comment
https://github.com/ptourlas/docs/commit/d6748f9495b2d5e78ef5801749e0dfe27cf5470b